modules/dashboard/README.md

```diff
@@ -0,0 +1,33 @@
+# Dashboard
+
+## Purpose
+
+The Dashboard serves as a landing page for all users logging into LORIS.  It provides introductory information and a primary quick-access interface for user workflows, including outstanding tasks and notifications of recent activity/additions in LORIS.  
+
+The Dashboard also provides at-a-glance live visual summaries of the data stored in LORIS, and dynamically plots data breakdowns that can be used in reporting. 
+
+## Intended Users
+
+All LORIS users can view the dashboard.
+
+## Scope
+
+The Dashboard provides entrypoints to various workflows, but does not provide any tools for directly modifying data in the dashboard. All users, regardless of site or project affiliation, can see data summaries and Tasks/Notifications across sites and projects. 
+
+The Study Progression chart views at bottom may or may not count `excluded` candidates, and this should be clearly verified and documented/presented for the use of your LORIS users. 
+
+## Permissions
+
+Components of the Dashboard are governed by permissions related to module-specific privileges. For example, the _Document Repository_ notifications panel is displayed if a user has been granted either the `document_repository_view` or `document_repository_delete` permission. 
+Site access and Project access privileges are not currently enforced in the display of data or notifications. 
```

Comment:
Are you sure about that? Why does the "My Tasks" box on the right list the site, then?

Reply:
I asked Christine to ask this based on this PR that is trying to add permissions to the site statistics.

https://github.com/aces/Loris/pull/5847

I don't know many details but it seems that the dashboard only partially enforces permissions right now.

Reply:
Maybe data do not care about permissions but notifications do?

Reply:
> Maybe data do not care about permissions but notifications do?

True, that's a notifications-specific issue.  I'll remove that mention, since it's not governed within the dashboard per se.